packages/admin-stories/src/admin/table/TableWithFilterBar.tsx

```diff
@@ -107,23 +107,24 @@ function Story() {
             <TableFilterFinalForm filterApi={filterApi}>
                 <Typography variant="h5">FilterBar</Typography>
                 <FilterBar>
-                    <FilterBar.Field label={"Username"} name="username">
+                    <FilterBar.Field label={"Username"}>
                         <Field name="username" type="text" component={FinalFormInput} fullWidth />
                     </FilterBar.Field>
                     <FilterBar.MoreFilters>
-                        <FilterBar.Field label={"Email"} name="email">
+                        <FilterBar.Field label={"Email"}>
                             <ExampleWithSelect />
                         </FilterBar.Field>
-                        <FilterBar.Field label={"Name"} name="name">
+                        <FilterBar.Field label={"Name"}>
                             <Field name="name" type="text" component={FinalFormInput} fullWidth />
                         </FilterBar.Field>
-                        <FilterBar.Field label={"Website"} name="website">
-                            <Field name="url" type="text" component={FinalFormInput} fullWidth />
-                            <Field name="name" type="text" component={FinalFormInput} fullWidth />
+                        <FilterBar.Field label={"Website"}>
+                            <Field name="website.url" type="text" component={FinalFormInput} fullWidth />
```

Comment:
Like this the name-attribute on Filterbar.Field is not needed, which is good.

But the user has now also the flexibility to mix fields of groups:
```
 <FilterBar.Field label={"Website"}>
          <Field name="website.url" type="text" component={FinalFormInput} fullWidth />
</FilterBar.Field>
 <FilterBar.Field label={"Something else"}>
          <Field name="website.name" type="text" component={FinalFormInput} fullWidth />
</FilterBar.Field>
```

Is this desired/acceptable?

Reply:
> Is this desired/acceptable?

yes, it's like in any other form where you group fields using eg. fieldsets

In pratice you won't use subfields that often I guess, I would name the filter exactly like the api does.